src/app/pages/homepage/homepage.html

```diff
@@ -6,60 +6,81 @@ <h1 class="mat-h1">Angular Material</h1>
+      </p>
+      <p>Straightforward APIs with consistent cross platform behaviour.</p>
     </div>
-    <div class="docs-homepage-promo-desc">
-      <h2>Sprint from Zero to App</h2>
-      <p>Hit the ground running with comprehensive, modern UI components that work across
-        the web, mobile, and desktop.</p>
-    </div>
-  </div>
-  <div class="docs-homepage-row docs-homepage-reverse-row">
-    <div class="docs-homepage-promo-img">
-      <docs-svg-viewer src="../assets/img/homepage/fastandconsistent.svg"
-                       [scaleToContainer]="true"></docs-svg-viewer>
-    </div>
-    <div class="docs-homepage-promo-desc">
-      <h2>Fast and Consistent</h2>
-      <p>Finely tuned performance, because every millisecond counts. Fully tested across
-        modern browsers.</p>
-    </div>
-  </div>
-  <div class="docs-homepage-row">
-    <div class="docs-homepage-promo-img">
-      <docs-svg-viewer src="../assets/img/homepage/versatile.svg"
-                       [scaleToContainer]="true"></docs-svg-viewer>
-    </div>
-    <div class="docs-homepage-promo-desc">
+    <div class="docs-homepage-row-column">
       <h2>Versatile</h2>
-      <p>Themeable, for when you need to stay on brand or just have a favorite color.
-        Accessible and internationalized so that all users are welcome.</p>
-    </div>
-  </div>
-  <div class="docs-homepage-row docs-homepage-reverse-row">
-    <div class="docs-homepage-promo-img">
-      <docs-svg-viewer src="../assets/img/homepage/optimized.svg"
-                       [scaleToContainer]="true"></docs-svg-viewer>
+      <p>
+        Provide tools that help developers build their own custom components with common interaction patterns.
+      </p>
+      <p>
+        Customizable within the bounds of the Material Design specification.
+      </p>
     </div>
-    <div class="docs-homepage-promo-desc">
-      <h2>Optimized for Angular</h2>
-      <p>Built by the Angular team to integrate seamlessly with Angular.</p>
+    <div class="docs-homepage-row-column">
+      <h2>Frictionless</h2>
+      <p>
+        Built by the Angular team to integrate seamlessly with Angular.
+      </p>
+      <p>
+        Start from scratch or drop into your existing applications.
+      </p>
     </div>
   </div>
-  <div class="docs-homepage-bottom-start">
-    <a mat-raised-button class="docs-button" routerLink="/guide/getting-started" color="accent">
-      Get started
+
+  <mat-divider></mat-divider>
+
+  <div class="docs-homepage-featured-components docs-homepage-carousel-row">
+    <h2>Featured components</h2>
+    <app-carousel [itemWidth]="260" [aria-label]="'Featured components'">
+      <a *ngFor="let comp of getTopComponents()" carousel-item class='docs-featured-components-carousel-item'
+         routerLink="/components/{{comp}}">
+        <div class="docs-homepage-img-container"><img alt="{{comp}}" src="../../../assets/screenshots/{{comp}}.scene.png"></div>
```

Comment:
Can you put `role="presentation"` on the image? Currently ChromeVox is reading everything twice, once for the image and once for the text. e.g. "button button"

Reply:
I tried putting `role="presentation"` but ChromeVox was still reading everything twice so I just removed the `alt` . I think this should be fine since it's same as the label below